.github/workflows/linux_meson.yml

```diff
@@ -248,3 +249,89 @@ jobs:
           export PYTHONOPTIMIZE=2
           python -m pytest --pyargs scipy -n2 --durations=10
           popd
+
+  #################################################################################
+  prerelease_deps_coverage_64bit_blas:
+    # TODO: re-enable ILP64 build.
+    name: Prerelease deps and 64-bit BLAS
+    # If using act to run CI locally the github object does not exist and the usual skipping should not be enforced
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9]
+
+    #    env:
+    #      NPY_USE_BLAS_ILP64: 1
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Setup Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Ubuntu dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libgmp-dev libmpfr-dev libmpc-dev ccache gfortran
+
+        # Install OpenBLAS a la cibuildwheel.
+        chmod +x tools/wheels/cibw_before_build_linux.sh
+        sudo tools/wheels/cibw_before_build_linux.sh .        
+
+    - name: Caching Python dependencies
+      uses: actions/cache@v3
+      id: cache
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-prerelease
+
+    - name: Install Python packages
+      run: |
+        # TODO: unpin cython and move to -pre once issues with cython 3.0
+        #       have been ironed out.
+        python -m pip install ninja meson meson-python wheel click rich_click pydevtool "cython==0.29.33"
+        python -m pip install --pre --upgrade --timeout=60 -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
+        python -m pip install --pre --upgrade --timeout=60 pytest pytest-xdist pybind11 mpmath gmpy2 pythran threadpoolctl pooch matplotlib
```

Comment:
this is PyPI, so the `--timeout=60` seems unnecessary

Reply:
```suggestion
        python -m pip install --pre --upgrade pytest pytest-xdist pybind11 mpmath gmpy2 pythran threadpoolctl pooch matplotlib
```